Makefile

```diff
@@ -95,3 +95,7 @@ git-commit-add:
 
 release-pod:
 	pod trunk push Sentry.podspec
+
+.PHONY: benchmark-sentry-pr
+benchmark-sentry-pr:
+	set -o pipefail && env NSUnbufferedIO=YES xcodebuild -workspace "Sentry.xcworkspace" -scheme "TrendingMovies-Benchmarking-UITests" -destination "OS=15.2,name=iPhone 13 Pro" test | xcpretty
```

Comment:
Maybe you want to store the raw test log into a file and upload it as an artifact if the tests fail. This is helpful when investigations flaky tests
https://github.com/getsentry/sentry-cocoa/blob/2d7cc7b287795b8d567643a870a4ef1067825dfe/scripts/xcode-test.sh#L68
https://github.com/getsentry/sentry-cocoa/blob/2d7cc7b287795b8d567643a870a4ef1067825dfe/.github/workflows/test.yml#L110-L116

Reply:
Thanks. My current issues aren't for a lack of logging 😅